components/base/FilterTabs.tsx

```diff
@@ -32,7 +32,7 @@ export default function Tabs({ tabs, filter, setFilter }: Props) {
       <TabsContainerStyle>
         <TabStyle
           className={`tab ${filter === null ? "is-active" : ""}`}
-          onClick={setFilter(null)}
+          onClick={() => setFilter(null)}
```

Comment:
Put the call to `setFilter` within a function here. Previously it would call `setFilter(null)` on every render and pass the return value to `onClick` instead of a function.